.gitignore

```diff
@@ -1,6 +1,6 @@
 node_modules/
-dist/
 .vscode/
+dist/
```

Comment:
had initially removed and changed my mind; Netlify can run our `build:prod` script before each deployment so no need to keep these files around